### ISSUE TITLE
Element in iplist removed all by itself

### ISSUE THREAD

**Issue by Fregf** [FoobarOy/foomuuri#56]
```
iplist {
	@localblocklist
}
```
and I added some element to it manually by running
`# foomuuri iplist add localblocklist xxx.xxx.xxx.xxx`

Today I suddenly noticed connections from an IP which I hadded added some time ago, where being allowed again. Later I checked with
`# foomuuri iplist list localblocklist`
and notice that that single IP was not in the list any more. Another IPv6 subnet which I recently added to this blocklist, was still there.

I re-added the IP again manually. Then I ran this:

```
# nft list set inet foomuuri localblocklist_4
table inet foomuuri {
	set localblocklist_4 {
		type ipv4_addr
		flags interval,timeout
		auto-merge
		elements = { xxx.xxx.xxx.xxx timeout 10d expires 9d22h16m4s232ms }
	}
}
```
Does this mean that the IP will be removed automatically 10 days after adding it? That's definitely not what I expected.

**Comment by kimheino** [comment]
> foomuuri iplist add localblocklist xxx.xxx.xxx.xxx

That uses iplist's timeout, which defaults to 10 days if not changed. So yes, it will expire. This was designed for fail2ban and similar use cases where timeout is needed.

You can specify timeout before IP address:

```
foomuuri iplist add localblocklist 365d 10.0.0.10
```

Personally I would use text file with comments why IP was added:

```
iplist {
    @localblocklist   /etc/foomuuri/my-block-list.txt    # don't use .conf here
}
```

```
# my-block-list.txt

# spammer, added 2024-12-01
10.0.0.10

# another spammer
10.0.0.20
```

I think this timeout-format is forgotten from documentation. I'll fix my documentation, not the code.

**Comment by Fregf** [comment]
It would be nice to have an option to disable automatic expiration.  This even interferes with Fail2ban. I have configured incremental banning ( https://visei.com/2020/05/incremental-banning-with-fail2ban/ ), and this way some IPs got banned for weeks or even months by Fail2ban. But then Foomuuri breaks this because it will unban them in practice after 10 days already.

**Comment by kimheino** [comment]
If one year expire is not enough, how about 10 years (3650 days):

```
foomuuri iplist add localblocklist 3650d 10.0.0.10
```

Or 100 years, `36500d`? That's basically same as disabled.

**Comment by Fregf** [comment]
Yes, but I'm not sure how efficient it is to keep track of the expiration time for more than 140000 elements for no reason, in my case.
